webextensions/manifest/incognito.json

```diff
@@ -74,8 +74,8 @@
                 "version_added": "17"
               },
               "firefox": {
-                "version_added": false,
-                "impl_url": "https://bugzil.la/1380812"
+                "version_added": "127",
+                "notes": "Split mode is not supported in Firefox. Extensions using this value are treated as requesting \"incognito\":\"not_allowed\"."
```

Comment:
Can you somehow link https://bugzilla.mozilla.org/show_bug.cgi?id=1380812 ? Whether from the mdn/content or BCD?

Can impl_url render meaningfully here?

This is effectively a partial_impl at best or just not supported.

Safari doesn't support incognito:split either, so their behavior is comparable to the default incognito:spanning mode.

Firefox was special as in it rejected values of incognito:split because it did recognize the incognito key and accepted only recognized values (spanning and not_allowed). Because the implementation rejected unrecognized values, incognito:split could not be installed. The change in 127 is to allow the extension to be installed, but to be overly cautious the default value is not_allowed.